R/ml_model_constructors.R

```diff
@@ -165,6 +165,20 @@ new_ml_model_clustering <- function(pipeline_model, formula, dataset,
   )
 }
 
+#' @export
```

Comment:
Let's not export these constructors since I don't see any other packages using them.